src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/Serialization/IDesignerSerializationManager.cs

```diff
@@ -71,30 +71,30 @@ public interface IDesignerSerializationManager : IServiceProvider
         /// of named instances. Objects that implement IComponent will be
         /// added to the design time container if addToContainer is true.
         /// </summary>
-        object CreateInstance(Type type, ICollection arguments, string name, bool addToContainer);
+        object CreateInstance(Type type, ICollection arguments, string? name, bool addToContainer);
```

Comment:
I am unsure if the `arguments` parameter should be nullable. The documentation does not specify it is nullable.

Reply:
Winforms has a null check against it, if that helps.

Reply:
Based on:
https://source.dot.net/#System.Windows.Forms.Design/System/ComponentModel/Design/Serialization/TypeCodeDomSerializer.cs,90
seems like it should be nullable.